dashboard/app/models/pd/local_summer_workshop_survey.rb

```diff
@@ -49,9 +51,11 @@ def self.public_required_fields
   end
 
   def self.required_fields
-    public_required_fields + [
-      :give_permission_to_quote
-    ].freeze
+    (
+      public_required_fields + [
+        :give_permission_to_quote
+      ]
+    ).freeze
```

Comment:
I only set it up this way for consistency and so it would be clear where to add more fields if we want. No strong reason where it fits.